### DISCUSSION TITLE
Communication mechanism between tracee-ebpf and trace-rules

### DISCUSSION THREAD

**Discussion by leiqi96** [aquasecurity/tracee#1101] [Development]
tracee-ebpf and tracee-rules are 2 processes that communicate over a pipe. Where should I start learning this code

**Comment by itaysk** [comment]
This would be in the entrypoint script. https://github.com/aquasecurity/tracee/blob/main/entrypoint.sh

**Reply by leiqi96** [reply]
Is the Linux pipe  FIFO? which code show that.

**Reply by itaysk** [reply]
Currently it's not a "Linux FIFO" but a simple shell pipe ("|") (but there's an open PR to make it a FIFO). 
You can see the code in the previous answer.
